packages/pwa-buildpack/src/WebpackTools/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
     MagentoRootComponentsPlugin: require('./plugins/MagentoRootComponentsPlugin'),
     ServiceWorkerPlugin: require('./plugins/ServiceWorkerPlugin'),
+    DevServerReadyNotifierPlugin: require('./plugins/DevServerReadyNotifierPlugin'),
```

Comment:
Unwieldy on purpose, yeah. PWADevServer should eventually turn from a factory `DevServerConfig => WebpackDevServerConfig` into a decorator `(WebpackConfig, DevServerConfig) => WebpackConfig`, and then it would apply this plugin without exposing its big unwieldy name.